payment-order-integration/pom.xml

```diff
@@ -71,24 +123,21 @@
             <artifactId>jackson-databind-nullable</artifactId>
             <version>${jackson-databind-nullable.version}</version>
         </dependency>
-        <dependency>
-            <groupId>com.fasterxml.jackson.datatype</groupId>
-            <artifactId>jackson-datatype-jsr310</artifactId>
-            <version>${jackson-datatype-jsr310.version}</version>
-        </dependency>
-
         <dependency>
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
             <version>${jsr305.version}</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.junit.vintage</groupId>
-            <artifactId>junit-vintage-engine</artifactId>
-            <scope>test</scope>
-        </dependency>
+        <!-- Add dependencies for your services, e.g. BB raml specifications, integration clients -->
 
+        <!-- Uncomment the following dependency if DBS inter-service communication is needed -->
```

Comment:
This comment & disabled dependency can be removed, as the dependency is listed above.

Reply:
corrected